### PR TITLE
Features/0.9.0

### PR DESCRIPTION
**Pull request by mbgspcii**
Apply to dev the correction "placeholder being visible under an image with translucency" + "subpixel bleed-through". 

That should fixe https://github.com/TwicPics/components/issues/11.

+ adds support for angular 14

+ update dependencies

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,10 +8,21 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
...
+
+- Add support for **Angular 14+**
+
 ### Changed
 
 - Upgrades dependencies and documentation.
 
+### Fixed
+
+- Fixe placeholder being visible under an image with translucency
```
Comment (Strift) [inline CHANGELOG.md:23]: Replace with: "Fix an issue that made the placeholder visible under some transparent images"

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,10 +8,21 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
...
+- Fixe subpixel bleed-through
```
Comment (Strift) [inline CHANGELOG.md:24]: Remove extra 'e' in "Fixe"

File: CHANGELOG.md (hunk 1 above)
Comment (mbgspcii) [reply CHANGELOG.md:23]: - [ ] Replace with: "Fix an issue that made the placeholder visible under some transparent images"

File: CHANGELOG.md (hunk 2 above)
Comment (mbgspcii) [reply CHANGELOG.md:24]: - [ ] Remove extra 'e' in "Fixe"

**Pull request merged**